README.md

```diff
@@ -1,15 +1,20 @@
-# Readit - Command line tool
+# Readit - Command Line Tool
 
 
 ## Introduction
-Readit is command line bookmark manager. It is a command line utility to add, delete, update the bookmarks. It is a powerful bookmark manager written in Python3 and SQLite3.
+Readit is command line bookmark manager. It is a command line utility to add, delete, update the bookmarks. It is a powerful bookmark manager written in Python and SQLite3.
```

Comment:
How about writing this as,
It is a powerful bookmark manager written in Python. It uses SQLite3 to store the bookmarks.

Reply:
@bhavin192 
Yes, It will be fine too. Also, I have introduced those utilities in features.

Reply:
I was talking about the last sentence only not the sentence containing small list of features. It is good to mention those features in first few lines.

Reply:
Yes, I understood. I will update introduction.